### ISSUE TITLE
Inconsistent bag icon size

### ISSUE THREAD

**Issue by henrikhermansen** [SpareBank1/designsystem#311]
#### Which package is this issue related to?
ffe-icons

#### Describe your issue (screenshots welcome!)
While PPoseIkon and KrPoseIkon has the same size, PPoseStjerneIkon appears a bit squashed.
I took some screen shots of PPoseIkon and PPoseStjerneIkon. PPose and KrPose would look the same.

While the icons should be the same size (as they are on open pages):
|![image](https://user-images.githubusercontent.com/435037/41778863-4ce04368-7630-11e8-94df-93f81e612837.png)|![image](https://user-images.githubusercontent.com/435037/41778857-49ee4c5e-7630-11e8-87c1-9c22725318bd.png)|

They appear differently in our design guide, and when they are used by a consumer:
|![image](https://user-images.githubusercontent.com/435037/41778908-6b57c938-7630-11e8-9e1f-332c9b2a88c4.png)|![image](https://user-images.githubusercontent.com/435037/41778998-b42a6b5c-7630-11e8-9b56-0fe4021a76fe.png)|

As you can see by the pipe I added after the images there's too much spacing on the sides of PPoseStjerneIkon. Its stroke is also too thin because it's been squashed.
It's also clear that the PPoseIkon is missing the space it should have above the bag.

Btw. the stroke seems correct on PPoseIkon as they are now. Compared to some other icon:
![image](https://user-images.githubusercontent.com/435037/41779198-58db7376-7631-11e8-8b49-d17e5972cd3f.png)

#### What is the expected behavior? 
Bag icons (*PoseIkon) should have the same bag size.

One could argue "icons should have the same viewbox and never have spacing around them" just to have a simple rule to follow when adding SVGs. This proves, however, that no rules are without exceptions.

As a consumer I expect icons should look correct out of the box. The PPoseStjerneIkon will never look correct, even if it's not placed next to other bag icons, because its stroke is too thin.

#### What is the actual behavior?
Bag icons have different bag sizes.

#### Steps to reproduce
Compare our [Design guide](http://design.sparebank1.no/styleguidist/index.html#ikoner) to the open banking pages for [pension](https://www.sparebank1.no/nb/smn/privat/sparing/pensjon/pensjonskonto.html) and [ips](https://www.sparebank1.no/nb/smn/privat/sparing/pensjon/ips.html).

**Comment by antidecaf** [comment]
Afaic, these icons probably need to work not only in the context of eachother but also with other icons. This means positioning the one with no star to fit the star might end up making it appear a bit off balance in the context of different icons, due to the extra whitespace within the viewbox. Example:

![ok](https://user-images.githubusercontent.com/463847/41780792-03ca1626-7636-11e8-9208-d6f980270ac8.png)

![wat](https://user-images.githubusercontent.com/463847/41780793-03e4e2d0-7636-11e8-9369-1080e6801706.png)

Thoughts, @adidick?

**Comment by antidecaf** [comment]
@henrikhermansen @adidick Any updates? Should we close this issue?

**Comment by adidick-zz** [comment]
I'm not sure what the best solution is here. 

@henrikhermansen I see you have used a 140x140px viewbox around the icons on the open pages. I'm assuming that's not always going to be possible in all use-cases where there is less space for the icon.

It also doesn't really make sense to have to have an exception built in to one icon, so I wonder if it would actually be of benefit to redesign the bag icon with the star over it? That seems to me to be the simplest and safest solution?

Thoughts?

**Comment by henrikhermansen** [comment]
@adidick my apologize for the late response.
Actually, I don't manage the open page, just the logged in ones. I can't recall whether I tried changing the viewbox or not, but I'm not sure I can when I'm using the icons from the react package.
I did create the issue for ffe-icons, but perhaps I should've made it for ffe-icons-react.
I'll have to look a bit into this again, and see what I did and didn't do.

Redesigning an icon or two, with this issue in mind, could indeed seem like a good idea.
